index.js

```diff
@@ -206,7 +206,7 @@ function checked (length) {
     throw new RangeError('Attempt to allocate Buffer larger than maximum ' +
                          'size: 0x' + kMaxLength.toString(16) + ' bytes')
   }
-  return length >>> 0
+  return length | 0
```

Comment:
Consistency?  I'm happy to try and merger this back up stream.
On 23 Apr 2015 7:52 am, "Feross Aboukhadijeh" <notifications@github.com>
wrote:

> In index.js
> <https://github.com/feross/buffer/pull/59#discussion_r28918646>:
>
> > @@ -206,7 +206,7 @@ function checked (length) {
> >      throw new RangeError('Attempt to allocate Buffer larger than maximum ' +
> >                           'size: 0x' + kMaxLength.toString(16) + ' bytes')
> >    }
> > -  return length >>> 0
> > +  return length | 0
>
> Why? I'd rather keep this the same as the iojs codebase so it's easier to
> stay in sync.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/feross/buffer/pull/59/files#r28918646>.
>